generic/tests/netperf.py

```diff
@@ -141,6 +144,25 @@ def mtu_set(mtu):
             process.run(host_mtu_cmd % (iface, mtu), ignore_status=False,
                         shell=True)
 
+    def tweak_tuned_profile():
+        """
+
+        Tweak configuration with truned profile
+
+        """
+
+        client_tuned_profile = params.get("client_tuned_profile")
+        server_tuned_profile = params.get("server_tuned_profile")
+        host_tuned_profile = params.get("host_tuned_profile")
+        error_context.context("Changing tune profile of guest", logging.info)
+        if params.get("os_type") == "linux":
+            ssh_cmd(server_ctl, server_tuned_profile)
+
+        error_context.context("Changing tune profile of client/host",
+                              logging.info)
+        ssh_cmd(client, client_tuned_profile)
+        process.run(host_tuned_profile, ignore_status=False, shell=True)
+
```

Comment:
Is the cmd run in the host? could we use ssh_cmd(host, host_tuned_profile)?

Reply:
I think do not need since process.run is for the host command.

Reply:
I think do not need since process.run is for the host command.

Reply:
I think do not need since process.run is for the host command.

Reply:
From Line 486-488, I think one purpose that you define ssh_cmd  is to run command in the host. So if you want to run host command, then you can directly use it.

Reply:
I have updated, please help a review.